caikit/core/module.py

```diff
@@ -786,14 +788,33 @@ def __init__(self, module: ModuleBase, model_path, library_name, library_version
             model_path:  str
                 The absolute path to the directory where the model will be saved.  If this directory
                 does not exist, it will be created.
-            library_name:  str
-                Name of library or extension calling the block saver.
-            library_version:  str
-                Version of library or extension calling the block saver.
         """
         self.model_path = os.path.normpath(model_path)
-        self.library_name = library_name
-        self.library_version = library_version
+
+        module_path = module.__module__
```

Comment:
Yeah, though maybe we were implementing more than we needed to.
I think the version override is required for users to save models correctly when running from source code, the module path stuff we threw in for fun because we had some old TODOs about it